programs/marginfi/src/state/marginfi_account.rs

```diff
@@ -314,6 +325,18 @@ impl<'a, 'b> BankAccountWithPriceFeed<'a, 'b> {
     }
 }
 
+enum PriceFeedError {
```

Comment:
Why is this not just another error variant in `MarginfiError`? Especially given that it's only one variant

Reply:
(I suspect this may make handling this error easier)

Reply:
Yes exactly, also the marginfi error gets converted into an AnchorError which is super bloated